lib/DllEntryPlugin.js

```diff
@@ -16,16 +16,18 @@ class DllEntryPlugin {
 	}
 
 	apply(compiler) {
-		compiler.plugin("compilation", (compilation, params) => {
+		compiler.hooks.compilation.tap("DllEntryPlugin", (compilation, { normalModuleFactory }) => {
 			const dllModuleFactory = new DllModuleFactory();
-			const normalModuleFactory = params.normalModuleFactory;
-
 			compilation.dependencyFactories.set(DllEntryDependency, dllModuleFactory);
-
 			compilation.dependencyFactories.set(SingleEntryDependency, normalModuleFactory);
 		});
-		compiler.plugin("make", (compilation, callback) => {
-			compilation.addEntry(this.context, new DllEntryDependency(this.entries.map((e, idx) => {
+
+		compiler.hooks.make.tapAsync("DllEntryPlugin", (compilation, callback) => {
+			const { context, entries } = this;
+
+			console.log(context, entries);
```

Comment:
🤕

Reply:
On it! 😅

Reply:
We should add an eslint for no consoles <3